aws-services/reference/supported-services.hbs.md

```diff
@@ -2,7 +2,9 @@
 
 This topic lists the services that are supported by the AWS Services package.
 
-## <a id="postgresql"></a> RDS PostgreSQL
+In Tanzu Application Platform v1.7, the AWS Services package only supports the following AWS services:
```

Comment:
Nitpick I would not say `only` here. "the AWS Services package supports the following AWS services" should be sufficient.

Reply:
I agree - fixed!